infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/PodTerminationGracePeriodProvisioner.java

```diff
@@ -60,4 +64,12 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity identity)
   private boolean isTerminationGracePeriodSet(final PodSpec podSpec) {
     return podSpec.getTerminationGracePeriodSeconds() != null;
   }
+
+  private long getGraceTerminationPeriodSec(KubernetesEnvironment k8sEnv) {
+    if (isEphemeral(k8sEnv.getAttributes())
```

Comment:
minor suggestion to use local var for `k8sEnv.getAttributes()` instead of calling it twice in the same if clause

Reply:
done